packages/client/hmi-client/src/components/model/model-parts/tera-model-part-entry.vue

```diff
@@ -111,22 +111,27 @@ import { CalendarDateType } from '@/types/common';
 
 const props = defineProps<{
 	item: ModelPartItem;
+	description?: string;
+	name?: string;
+	unitExpression?: string;
+	templateId?: string;
+	id?: string;
 	featureConfig: FeatureConfig;
```

Comment:
Maybe I am missing something here, beforehand it is `props.item.description`, and now it is `props.description`. Why does it matter if it is nested under props or nested under props.item?

Reply:
Better reactivity. I don't even think we need the `item` anymore.

Reply:
In terms of reactivity, I think it might be a incorrect setup in Vue and we can address that. I don't know what to make of whether we need "item" or not, I did not take an in-depth read.

So we have this (or something similar to this):
```
<tera-model-part-entry
  v-else
  :is-time-part="!!isTimePart"
  :item="base"
  :description="base.description"
  :name="base.name"
  :id="base.id"
  :templateId="base.templateId"
  :unitExpression="base.unitExpression"
  :feature-config="featureConfig"
  @update-item="$emit('update-item', { id: base.id, ...$event })"
/>
```

The issue is that we have:
- `props.item.descriptions` and `props.description`
- `props.item.name` and `props.name`
- ...etc

That is both redundant and error-prone. I think if we need sub-items to be reactive, either
```
const description = ref(props.item.description)
```

or

```
const itemRef = ref(props.item)
```

or, as Yohann said if we don't need to governing "item" and just pass all the fields in as first-level props.